File: HaoConnect/ios/HaoConnect/HaoConfig-example.m
//
//  HaoConfig.m
//  HaoxiHttprequest
//
//  Created by lianghuigui on 15/11/29.
//  Copyright © 2015年 lianghuigui. All rights reserved.
//

#import "HaoConfig.h"

@implementation HaoConfig
+(NSString *)getClientInfo{
    
    return @"???";
}

+(NSString *)getSecretHax{
    
    return @"secret=???";
}

+(NSString *)getApiHost{
    
    return @"api.???.com";
    
}
+(NSString *)getClientVersion{
    
    return [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
}
@end

File: HaoConnect/android/HaoConnect/src/main/java/com/haoxitech/HaoConnect/HaoConfig_example.java
package com.haoxitech.HaoConnect;

/**
 * Created by wangtao on 15/12/21.
 */
public class HaoConfig_example {

    private static String Clientversion = "";

    public static String getClientInfo() {
        return "???";
    }

    public static void setClientVersion(String clientVersion) {
        Clientversion = clientVersion;
    }

    public static String getClientVersion() {
        return Clientversion;
    }

    public static String getSecretHax() {
        return "secret=???";
    }

    public static String getApiHost() {
        return "api.???.com";
    }
}
